rest-api/dao/participant_dao.py

```diff
@@ -114,7 +114,7 @@ def _do_update(self, session, obj, existing_obj):
       need_new_summary = True
 
     if obj.organizationId or obj.siteId or obj.hpoId:
-      site, organization, awardee = self.get_pairing_level(obj, existing_obj)
+      site, organization, awardee = self.get_pairing_level(obj)
```

Comment:
This change looks like it will work functionally.

In future, we may want to consider not calling _do_update() from add_missing_hpo_from_site(), and instead calling a simpler function that just messes with site, organization, awardee, and lastModified, since those are the only things that should change. (It can call helper functions to borrow common functionality.)

Reply:
agreed.